javascript/javascript1/week1/housepricecalculator.js

```diff
@@ -0,0 +1,29 @@
+let volumeInMeters;
+let housePrice;
+// Peters house price calculation 
+let gardenSizeInM2 = 100;
+let width = 8,
+    depth = 10,
+    height = 10;
+let actualHousePrice = 2500000;
+volumeInMeters = width * depth * height;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+if (actualHousePrice > housePrice) {
+    console.log("Peter is paying too much");
+} else {
+    console.log("Peter is paying reasonable");
+}
+// Julia house price calculation
```

Comment:
here, in this code you could use arrays to assign the values instead of re-assigning. like this(const width=[8,5],
 depth=[10,11],
height=[10,8];). then you can compare with their index value (like: if(actualHousePrice[0] > housePrice[0]){
    console.log("expensive house");
})